var/spack/repos/builtin/packages/py-geant4/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyGeant4(CMakePackage):
+    """Python bindings for Geant4 (g4py)"""
+
+    homepage = "http://geant4.cern.ch/"
+    git = "https://gitlab.cern.ch/geant4/geant4.git"
+    url = "https://gitlab.cern.ch/geant4/geant4/-/archive/v10.6.0/geant4-v10.6.0.tar.gz"
```

Comment:
```suggestion
    url = "https://gitlab.cern.ch/geant4/geant4/-/archive/v10.6.1/geant4-v10.6.1.tar.gz"
```
It doesn't make a difference for fetching, but our URL accuracy is based on how many versions that we extract from a URL are actually listed in the package.